api/prisma/schema.prisma

```diff
@@ -145,6 +145,145 @@ model user {
 
 // -----------------------------------
 
+/// An exam as designed by the examiners
+model NewExam {
+  /// Globally unique exam id
+  ///
+  /// NOTE: Field is not "default", because of when/where exams are created.
+  id                 String         @id @map("_id") @db.ObjectId
+  /// Human-readable exam name
+  name               String
+  /// Notes given about exam
+  accessibility_note String
+  /// All questions for a given exam
+  question_types     QuestionType[]
+  /// Configuration for exam metadata
+  config             NewConfig
+}
+
+type QuestionType {
+  /// Unique question type id
+  id        String          @db.ObjectId
+  type      NewQuestionType
+  text      String?
```

Comment:
`text` is very generic. What is this for?

Reply:
It is the question text... ?

Reply:
Right, but "text" is implied because it's a string! What kind of text is it? A description? A name?

Reply:
It is whatever the exam maker wants. It can be a question, it can be a description, it can be an instruction, it can be added context...

Reply:
Surely those should be different properties.

Reply:
Well, no. They are all just text that appear before the question text itself.

Reply:
At the moment the `question_type` only holds text if it is a dialogue type. Why not call the attribute `dialogue_text` for now?

Paragraph could also be an option.

Reply:
> Why not call the attribute dialogue_text for now?

Because we cannot change it in the future. Also, it is not a given it will just be fore dialogue types. We discussed adding text for non-dialogues to it as well for things like audio questions.

Reply:
We can make it more descriptive by calling it description instead. It describes what the question is about.

Reply:
Best I would suggest is `context`. `description` does not work for dialogue text.

Reply:
That sounds reasonable, I quite like it.